examples/gno.land/p/demo/markdown/markdown_test.gno

```diff
@@ -0,0 +1,118 @@
+package markdown
```

Comment:
This package is a perfect example of something that would benefit from a realm-based demo `r/demo/md`. The demo should iterate over the available helpers, displaying their usage and the corresponding results.

Reply:
renamed to `md`